src/js/components/CheckBox/StyledCheckBox.js

```diff
@@ -89,7 +92,10 @@ const StyledCheckBoxToggle = styled.span`
       ? normalizeColor(props.theme.checkBox.toggle.background, props.theme)
       : 'transparent'};
 
-  ${props => props.focus && focusStyle()};
+  ${props =>
+    props.focusIndicator === false
+      ? props.focus && props.theme.checkBox.focusIndicator && focusStyle()
```

Comment:
It looks like theme value is overriding props here. is that true? I would anticipate if focusIndicator is false, then focusStyle should not be applied.

Reply:
I agree that this logic is confusing. It gets convoluted trying to maintain backwards compatibility.

Ideally this line would read
```
${props => props.focus && props.focusIndicator !== false && focusStyle()};
```
but then we lose backwards compatibility. In the grommet theme, currently both the checkbox and the formfield bottom border get focus styling:
![Screen Shot 2021-02-05 at 9 03 48 AM](https://user-images.githubusercontent.com/1756948/107057902-2d212d80-6791-11eb-82e0-5751075567b4.png)
but if we implement as shown above, we lose the focus around the checkbox and only the formfield bottom border receives the treatment:
![Screen Shot 2021-02-05 at 9 04 09 AM](https://user-images.githubusercontent.com/1756948/107058126-71acc900-6791-11eb-8923-dab0365ab047.png)

So, if we want to avoid this, then we need some sort of opt in mechanism. So, this is trying to say:
"IF focusIndicator is false, THEN check whether I have opted into this behavior." I will play around with alternatives, but open to ideas if you have any.

Reply:
I would be OK with the grommet theme behavior having the focus indicator only on the bottom line, as I think that fits with the general behavior of focus in a form context. You could argue that the current behavior is a bug, being inconsistent.

Reply:
Yes, I had thought that as well, but wanted to bring visibility and discussion.

If we adapt that view, it eliminates the confusing aspects here. Proceed with the "dual focus is a bug" viewpoint? If so, I will clean up.

Reply:
@ShimiSun , would appreciate your input on whether we should look at the focus treatment of a CheckBox in a FormFiled as a bug since it is inconsistent with the treatment for other inputs? Or, whether we should maintain the current treatment?

Reply:
I think that the current behavior of the theme is a bug and I'm OK fixing it without worrying about the backward compatibility aspect -- in case it affects only the HPE theme users. 
Few questions:
1. Is this issue also reproducible for a single RadioButton outside of Form content?
2. Would there be any b/c issues for grommet itself not in the context of HPE?
3. Would the `focusIndicator` behavior be consistent with other input elements when used with `plain`?

Reply:
> 1. Is this issue also reproducible for a single RadioButton outside of Form content?
RadioButton has its own focus issues which should be looked at. For example noticed how focus is inconsistently applied in [this example](https://storybook.grommet.io/?path=/story/input-radiobuttongroup-simple--simple).

> 2. Would there be any b/c issues for grommet itself not in the context of HPE?
The backwards compatibility vs. bug question is being raised because it does affect Grommet users not in the context of HPE. That is the decision which needs to be made in order to inform what direction this PR takes.

> 3. Would the `focusIndicator` behavior be consistent with other input elements when used with `plain`?
That is the intent.

Reply:
I might be an idealist but I would love to leverage the final approach of the fix to fix RadioButton as well so this type of fixes won't be spread across different releases. Regarding the link you have attached of RadioButton it looks like the focus is displayed, so I'm not sure what you are pointing at that I might be missing.

I would like us to have a quick chat about the b/c issue and how it affects grommet users (tomorrow morning?), since the current reported issue is only to HPE users I wonder how far we should go with risking grommet b/c to fix it, hence I would like a discussion there. 

Also, it will be nice to see how the new approach is working with `plain`.

Reply:
I'm looking at the RadioButton example linked. I have not looked into the component implementation, but what I see behaviorally is:
1. I tab to the component, but there's no focus indication that I'm on it.
2. I press down arrow and a focus indicator appears on the selected RadioButton.
3. I press down arrow again and the next RadioButton gets selected but no focus indicator is shown.
4. I press down arrow again and the focus indicator appears again.

Overall, getting this back and forth behavior where focus indicator is shown, then removed, then shown again.

I would hope that Matt's work on Checkbox could be leveraged in RadioButton, but it also feels like there are some other funky focus behavior issues at play in RadioButtonGroup. Might take a little bit of separate investigation to understand.